src/components/button/button_toggle/button_toggle.test.tsx

```diff
@@ -8,6 +8,7 @@ describe('EuiButtonToggle', () => {
   test('is rendered', () => {
     const component = render(
       // @ts-ignore @chandler
+      // `onClick` is marked as required in EuiButtonTogglePropsForButtonToggle
       <EuiButtonToggle {...requiredProps} label="Label me" />
```

Comment:
`onClick` is required as defined by

```
type EuiButtonTogglePropsForButtonToggle = EuiButtonToggleProps &
  ButtonHTMLAttributes<HTMLButtonElement> & {
    onClick: MouseEventHandler<HTMLButtonElement>;
    value?: string;
  };
```

This same configuration affects `EuiButtonIcon` and its tests

```
type EuiButtonIconPropsForButton = EuiButtonIconProps &
  ButtonHTMLAttributes<HTMLButtonElement> & {
    onClick: MouseEventHandler<HTMLButtonElement>;
    buttonRef?: Ref<HTMLButtonElement>;
  };
```